vagrant/provisioning/roles/nmos-auth-install/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# module: nmos-auth-install/tasks
+# description: Install NMOS Auth Server
+
+- name: Move bin file
```

Comment:
Does the pip install not put this one in place? It may be the same could be achieved with a symlink if the bin file just gets installed in the wrong location.

Reply:
Pip is meant to, but when run on source code it places it in the following path
/usr/local/lib/python2.7/dist-packages/nmosauth/usr/bin/
instead of /usr/bin/
Will look at if this can be fixed

Reply:
If not can we just symlink to that location from /usr/bin/?

Reply:
Yeah will do that, if not

Reply:
If you run the following command everything installs where it should:
`sudo pip install --no-binary nmos-auth nmos-auth`

Reply:
The Ansible Pip module has an `extra-args` argument for you to pass in extra flags